_tsinfermodule.c

```diff
@@ -274,34 +275,6 @@ AncestorBuilder_ancestor_descriptors(AncestorBuilder *self)
         }
         PyTuple_SET_ITEM(descriptors, j, py_descriptor);
     }
-    /* j = 0; */
```

Comment:
I assume this is commented out as it might be useful for future debugging?

Reply:
This is all deleted , just old crufr that was commented out